lib/CONFIG/config.h

```diff
@@ -35,7 +35,7 @@
 //ESP32
 #else
 
-#define PIN_LED 21
+#define PIN_LED 16 //was 21
```

Comment:
could you add a separate target that supports the OLED display instead of modifying an existing one? Or add guards like :
```
#ifdef ESP32_OLED
  #define PIN_LED 16 //was 21
#else
  #define PIN_LED 21
#endif
```